pkgs/development/python-modules/gradio/default.nix

```diff
@@ -0,0 +1,204 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, writeText
+
+# pyproject
+, hatchling
+, hatch-requirements-txt
+, hatch-fancy-pypi-readme
+
+# runtime
+, aiofiles
+, aiohttp
+, altair
+, fastapi
+, ffmpy
+, markdown-it-py
+, mdit-py-plugins
+, markupsafe
+, matplotlib
+, numpy
+, orjson
+, pandas
+, pillow
+, pycryptodome
+, python-multipart
+, pydub
+, pyyaml
+, requests
+, uvicorn
+, jinja2
+, fsspec
+, httpx
+, pydantic
+, websockets
+, typing-extensions
+
+# check
+, pytestCheckHook
+, pytest-asyncio
+, mlflow
+, huggingface-hub
+, transformers
+, wandb
+, respx
+, scikit-image
+, shap
+, ipython
+, ffmpeg
+, vega_datasets
+, boto3
+}:
+
+let
+  # breaks way too frquently
+  shapWorks = (builtins.tryEval shap.outPath).success;
+in
+
```

Comment:
that's rather hacky, I would like to avoid this

Reply:
Shape was the reason the prior pr stalled

Reply:
We could just disable the tests all the time to avoid this. I am not even sure if tryEval has implications.

Reply:
I've removed those lines completely and disabled by default the problematic test.